bitcash/format.py

```diff
@@ -134,21 +116,24 @@ def wif_checksum_check(wif):
 
 
 def public_key_to_address(public_key, version='main'):
-    if version == 'test':
-        version = 'P2PKH-TESTNET'
-    elif version == 'regtest':
-        version = 'P2PKH-REGTEST'
-    elif version == 'main':
-        version = 'P2PKH'
-    else:
-        raise ValueError('Invalid version.')
+    # Currently Bitcash only support P2PKH (not P2SH)
+    VERSIONS = {
+        'main': "P2PKH",
+        'test': "P2PKH-TESTNET",
+        'regtest': "P2PKH-REGTEST"
+    }
+
+    try:
+        version = VERSIONS[version]
+    except:
```

Comment:
I think flake8 won't like this. This should be `except Exception:` at a minimum, but probably `except KeyError:`.